src/lib/proxy-server.ts

```diff
@@ -81,6 +82,33 @@ export async function startProxyServer(): Promise< boolean > {
 			} );
 		} );
 
+		// On Windows, we need to check if the port is in use before we can listen on it
+		if ( process.platform === 'win32' ) {
+			await new Promise< void >( ( resolve, reject ) => {
+				const tester = createConnection( { port: 80 }, () => {
+					// If we can connect, port is in use
+					tester.end();
+					const error = new Error( 'Port 80 is in use' ) as NodeJS.ErrnoException;
+					error.code = 'EADDRINUSE';
+					reject( error );
```

Comment:
```suggestion
					tester.end();
					reject( new Error( 'EADDRINUSE' ) );
```
`as` statements always irk me, even in relatively harmless cases like this. This is ultimately subjective, but I would advise changing this code per my suggestion and updating the catch clause to something like this:

```ts
	} catch ( error ) {
		if (
			( isErrnoException( error ) && error.code === 'EADDRINUSE' ) ||
			( error instanceof Error && error.message === 'EADDRINUSE' )
		) {
```